validator/client/validator.go

```diff
@@ -98,12 +102,15 @@ func (v *validator) WaitForActivation(ctx context.Context) error {
 		if err != nil {
 			return fmt.Errorf("could not receive validator activation from stream: %v", err)
 		}
-		validatorActivatedRecord = res.Validator
+		validatorActivatedRecords = res.ActivatedPublicKeys
 		break
 	}
-	log.WithFields(logrus.Fields{
-		"activationEpoch": validatorActivatedRecord.ActivationEpoch - params.BeaconConfig().GenesisEpoch,
-	}).Info("Validator activated")
+	for _, pk := range validatorActivatedRecords {
+		log.WithFields(logrus.Fields{
+			"public key": pk,
```

Comment:
```suggestion
			"public key": fmt.Sprintf("%#x", pk),
```
Otherwise it looks like
```
time="2019-04-04 17:58:25" level=info msg="Validator activated" prefix=validator public key=[131 3 163 192 108 250 67 155 105 246 122 85 3 169 146 17 236 226 147 86 192 18 90 65 255 39 223 223 43 46 236 39 241 67 48 251 116 56 211 118 23 162 6 94 219 6 216 193 16 183 145 6 255 20 2 173 15 68 75 186 163 116 254 72 11 82 157 27 80 14 44 223 87 171 115 127 96 130 147 178 77 89 177 190 148 138 80 78 212 156 227 66 134 180 244 113]
```

Reply:
thanks